### ISSUE TITLE
Change `gcp-postgresql|mysql-ssl` projects to create final connection string via .env file instead of providing via env var already

### ISSUE THREAD

**Issue by janpio** [prisma/ecosystem-tests#2012]
Right now a final connection string like `mysql://<user>:<password>@<host>:<port>/<database>?sslmode=require&sslcert=../server-ca.pem&sslidentity=../client-identity.p12&sslpassword=<client-identity-password>&sslaccept=accept_invalid_certs` is provided via env var. 

It would be more educational (and also easier to test variants) if we would instead only provide the default connection string via env var, and then used the .env file to build the final connection string.

**Comment by millsp** [comment]
~~Does it mean that we would do env var concats? Because I think `dotenv` does not support that. In that case, we could easily add `dotenv-expand`.~~

Ah nevermind, I just saw that we do have [our own version of `expand`](https://github.com/prisma/prisma/blob/main/packages/sdk/src/dotenvExpand.ts)

**Comment by janpio** [comment]
Yep, also documented at https://www.prisma.io/docs/guides/development-environment/environment-variables#expanding-variables
